### ISSUE TITLE
Plugin "rejected" not working

### ISSUE THREAD

**Issue by shinjidf** [Flexget/Flexget#3456]
flexget rejected list

### Expected behaviour:

lists rejected torrents

### Actual behaviour:
┏━━━┳━━━━━━━┳━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━┓
┃ # ┃ Title ┃ Task ┃ Rejected by ┃ Reason ┃
┡━━━╇━━━━━━━╇━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━┩
└───┴───────┴──────┴─────────────┴────────┘
empty list

### Steps to reproduce:
flexget rejected list
flexget rejected clear
flexget execute
flexget rejected list

#### Config:
```yaml
templates:
  global:
    verify_ssl_certificates: yes
    free_space:
      path: ****************
      space: 500
    magnets: yes
    exists_series:
      - '************'
      - '************'
    transmission:
      host: localhost
      port: 9091
      username: ************
      password: ************
    inputs:
      - rss: { url: 'https://subsplease.org/rss/?t&r=1080', silent: yes }
  anime1:
    inputs:
      - rss: { url: 'https://nyaa.si/?page=rss&q=komi-san+ember&c=1_2&f=0', silent: yes }
    series:
        - Tate no Yuusha no Nariagari S2
        - Dance Dance Danseur
        - Komi-san wa, Comyushou desu
        - Ao Ashi        
        - Gaikotsu Kishi-sama, Tadaima Isekai e Odekakechuu
        - Otome Game Sekai wa Mob ni Kibishii Sekai desu
  anime2:
    series:
        - Aharen-san wa Hakarenai
        - Deaimon
        - Honzuki no Gekokujou
        - Love All Play
        - Mahoutsukai Reimeiki
        - Paripi Koumei
        - Shijou Saikyou no Daimaou, Murabito A ni Tensei suru
        - Spy x Family
  anime3:
    series:
        - Kawaii dake ja Nai Shikimori-san
        - RPG Fudousan
        - Shachiku-san wa Youjo Yuurei ni Iyasaretai
        - Yatogame-chan Kansatsu Nikki S4
tasks:
  anime1:
    priority: 10
    template: anime1
  anime2:
    priority: 20
    template: anime2
  anime3:
    priority: 30
    template: anime3
```

Log
2022-06-01 23:50:30 VERBOSE  task_queue                    There are 3 tasks to execute. Shutdown will commence when they have completed.
2022-06-01 23:50:33 VERBOSE  details       anime1          Produced 72 entries.
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E08 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E08 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E07 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E07 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E06 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E06 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E05 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E05 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E04 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E04 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E03 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E03 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E02 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E02 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu S02E01 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu S02E01 [1080p] [HEVC WEBRip] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E12 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E12 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E11 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E11 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E10 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E10 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E09 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E09 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E08 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E08 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E07 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E07 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E06 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E06 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E05 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E05 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E04 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E04 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E03 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate) V3` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E03 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate) V3` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E02 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E02 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:33 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. S01E01 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` by seen plugin because entry with title `[EMBER] Komi-san wa, Comyushou desu. S01E01 [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate)` is already marked seen in the task anime1 at 2022-06-01 23:16
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Deaimon - 09 (1080p) [17ACD141].mkv` by seen plugin because entry with title `[SubsPlease] Deaimon - 09 (1080p) [17ACD141].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] RPG Fudousan - 09 (1080p) [C269EB4F].mkv` by seen plugin because entry with title `[SubsPlease] RPG Fudousan - 09 (1080p) [C269EB4F].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Tate no Yuusha no Nariagari S2 - 09 (1080p) [8050E843].mkv` by seen plugin because entry with title `[SubsPlease] Tate no Yuusha no Nariagari S2 - 09 (1080p) [8050E843].mkv` is already marked seen in the task anime1 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Shijou Saikyou no Daimaou, Murabito A ni Tensei suru - 09 (1080p) [5F131C9B].mkv` by seen plugin because entry with title `[SubsPlease] Shijou Saikyou no Daimaou, Murabito A ni Tensei suru - 09 (1080p) [5F131C9B].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Yuusha, Yamemasu - 09 (1080p) [AB314409].mkv` by seen plugin because entry with title `[SubsPlease] Yuusha, Yamemasu - 09 (1080p) [AB314409].mkv` is already marked seen in the task anime2 at 2022-05-31 23:33
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Honzuki no Gekokujou - 34 (1080p) [361A7367].mkv` by seen plugin because entry with title `[SubsPlease] Honzuki no Gekokujou - 34 (1080p) [361A7367].mkv` is already marked seen in the task anime2 at 2022-05-30 23:20
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Otome Game Sekai wa Mob ni Kibishii Sekai desu - 09 (1080p) [B9B04B93].mkv` by seen plugin because entry with title `[SubsPlease] Otome Game Sekai wa Mob ni Kibishii Sekai desu - 09 (1080p) [B9B04B93].mkv` is already marked seen in the task anime2 at 2022-05-29 23:39
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Kawaii dake ja Nai Shikimori-san - 07 (1080p) [CA7E54B2].mkv` by seen plugin because entry with title `[SubsPlease] Kawaii dake ja Nai Shikimori-san - 07 (1080p) [CA7E54B2].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Kakkou no Iinazuke - 06 (1080p) [C90ADCC1].mkv` by seen plugin because entry with title `[SubsPlease] Kakkou no Iinazuke - 06 (1080p) [C90ADCC1].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Spy x Family - 08 (1080p) [E1866698].mkv` by seen plugin because entry with title `[SubsPlease] Spy x Family - 08 (1080p) [E1866698].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Yatogame-chan Kansatsu Nikki S4 - 08 (1080p) [4EA5C260].mkv` by seen plugin because entry with title `[SubsPlease] Yatogame-chan Kansatsu Nikki S4 - 08 (1080p) [4EA5C260].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Ao Ashi - 08 (1080p) [21B0C543].mkv` by seen plugin because entry with title `[SubsPlease] Ao Ashi - 08 (1080p) [21B0C543].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Love All Play - 09 (1080p) [EBDD0D99].mkv` by seen plugin because entry with title `[SubsPlease] Love All Play - 09 (1080p) [EBDD0D99].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Dance Dance Danseur - 08 (1080p) [38C312D3].mkv` by seen plugin because entry with title `[SubsPlease] Dance Dance Danseur - 08 (1080p) [38C312D3].mkv` is already marked seen in the task anime1 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Aharen-san wa Hakarenai - 09 (1080p) [97D8A57D].mkv` by seen plugin because entry with title `[SubsPlease] Aharen-san wa Hakarenai - 09 (1080p) [97D8A57D].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Legend of the Galactic Heroes - Die Neue These - 35 (1080p) [C8C3D69C].mkv` by seen plugin because entry with title `[SubsPlease] Legend of the Galactic Heroes - Die Neue These - 35 (1080p) [C8C3D69C].mkv` is already marked seen in the task anime2 at 2022-05-26 23:54
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[SubsPlease] Mahoutsukai Reimeiki - 07 (1080p) [9356CBBD].mkv` by seen plugin because entry with title `[SubsPlease] Mahoutsukai Reimeiki - 07 (1080p) [9356CBBD].mkv` is already marked seen in the task anime2 at 2022-05-26 23:54
2022-06-01 23:50:34 VERBOSE  task          anime1          REJECTED: `[EMBER] Komi-san wa, Comyushou desu. (2021) (Season 1) [1080p] [HEVC 10 bits WEBRip DDP] (Komi Can't Communicate) (Batch)` by series plugin because season pack support is turned off
2022-06-01 23:50:34 VERBOSE  details       anime1          Summary - Accepted: 0 (Rejected: 38 Undecided: 34 Failed: 0)
2022-06-01 23:50:34 VERBOSE  input_cache   anime2          Restored entries from cache
2022-06-01 23:50:34 VERBOSE  details       anime2          Produced 50 entries.
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Deaimon - 09 (1080p) [17ACD141].mkv` by seen plugin because entry with title `[SubsPlease] Deaimon - 09 (1080p) [17ACD141].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] RPG Fudousan - 09 (1080p) [C269EB4F].mkv` by seen plugin because entry with title `[SubsPlease] RPG Fudousan - 09 (1080p) [C269EB4F].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Tate no Yuusha no Nariagari S2 - 09 (1080p) [8050E843].mkv` by seen plugin because entry with title `[SubsPlease] Tate no Yuusha no Nariagari S2 - 09 (1080p) [8050E843].mkv` is already marked seen in the task anime1 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Shijou Saikyou no Daimaou, Murabito A ni Tensei suru - 09 (1080p) [5F131C9B].mkv` by seen plugin because entry with title `[SubsPlease] Shijou Saikyou no Daimaou, Murabito A ni Tensei suru - 09 (1080p) [5F131C9B].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Yuusha, Yamemasu - 09 (1080p) [AB314409].mkv` by seen plugin because entry with title `[SubsPlease] Yuusha, Yamemasu - 09 (1080p) [AB314409].mkv` is already marked seen in the task anime2 at 2022-05-31 23:33
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Honzuki no Gekokujou - 34 (1080p) [361A7367].mkv` by seen plugin because entry with title `[SubsPlease] Honzuki no Gekokujou - 34 (1080p) [361A7367].mkv` is already marked seen in the task anime2 at 2022-05-30 23:20
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Otome Game Sekai wa Mob ni Kibishii Sekai desu - 09 (1080p) [B9B04B93].mkv` by seen plugin because entry with title `[SubsPlease] Otome Game Sekai wa Mob ni Kibishii Sekai desu - 09 (1080p) [B9B04B93].mkv` is already marked seen in the task anime2 at 2022-05-29 23:39
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Kawaii dake ja Nai Shikimori-san - 07 (1080p) [CA7E54B2].mkv` by seen plugin because entry with title `[SubsPlease] Kawaii dake ja Nai Shikimori-san - 07 (1080p) [CA7E54B2].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Kakkou no Iinazuke - 06 (1080p) [C90ADCC1].mkv` by seen plugin because entry with title `[SubsPlease] Kakkou no Iinazuke - 06 (1080p) [C90ADCC1].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Spy x Family - 08 (1080p) [E1866698].mkv` by seen plugin because entry with title `[SubsPlease] Spy x Family - 08 (1080p) [E1866698].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Yatogame-chan Kansatsu Nikki S4 - 08 (1080p) [4EA5C260].mkv` by seen plugin because entry with title `[SubsPlease] Yatogame-chan Kansatsu Nikki S4 - 08 (1080p) [4EA5C260].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Ao Ashi - 08 (1080p) [21B0C543].mkv` by seen plugin because entry with title `[SubsPlease] Ao Ashi - 08 (1080p) [21B0C543].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Love All Play - 09 (1080p) [EBDD0D99].mkv` by seen plugin because entry with title `[SubsPlease] Love All Play - 09 (1080p) [EBDD0D99].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Dance Dance Danseur - 08 (1080p) [38C312D3].mkv` by seen plugin because entry with title `[SubsPlease] Dance Dance Danseur - 08 (1080p) [38C312D3].mkv` is already marked seen in the task anime1 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Aharen-san wa Hakarenai - 09 (1080p) [97D8A57D].mkv` by seen plugin because entry with title `[SubsPlease] Aharen-san wa Hakarenai - 09 (1080p) [97D8A57D].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Legend of the Galactic Heroes - Die Neue These - 35 (1080p) [C8C3D69C].mkv` by seen plugin because entry with title `[SubsPlease] Legend of the Galactic Heroes - Die Neue These - 35 (1080p) [C8C3D69C].mkv` is already marked seen in the task anime2 at 2022-05-26 23:54
2022-06-01 23:50:34 VERBOSE  task          anime2          REJECTED: `[SubsPlease] Mahoutsukai Reimeiki - 07 (1080p) [9356CBBD].mkv` by seen plugin because entry with title `[SubsPlease] Mahoutsukai Reimeiki - 07 (1080p) [9356CBBD].mkv` is already marked seen in the task anime2 at 2022-05-26 23:54
2022-06-01 23:50:34 VERBOSE  details       anime2          Summary - Accepted: 0 (Rejected: 17 Undecided: 33 Failed: 0)
2022-06-01 23:50:34 VERBOSE  input_cache   anime3          Restored entries from cache
2022-06-01 23:50:34 VERBOSE  details       anime3          Produced 50 entries.
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Deaimon - 09 (1080p) [17ACD141].mkv` by seen plugin because entry with title `[SubsPlease] Deaimon - 09 (1080p) [17ACD141].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] RPG Fudousan - 09 (1080p) [C269EB4F].mkv` by seen plugin because entry with title `[SubsPlease] RPG Fudousan - 09 (1080p) [C269EB4F].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Tate no Yuusha no Nariagari S2 - 09 (1080p) [8050E843].mkv` by seen plugin because entry with title `[SubsPlease] Tate no Yuusha no Nariagari S2 - 09 (1080p) [8050E843].mkv` is already marked seen in the task anime1 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Shijou Saikyou no Daimaou, Murabito A ni Tensei suru - 09 (1080p) [5F131C9B].mkv` by seen plugin because entry with title `[SubsPlease] Shijou Saikyou no Daimaou, Murabito A ni Tensei suru - 09 (1080p) [5F131C9B].mkv` is already marked seen in the task anime2 at 2022-06-01 23:12
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Yuusha, Yamemasu - 09 (1080p) [AB314409].mkv` by seen plugin because entry with title `[SubsPlease] Yuusha, Yamemasu - 09 (1080p) [AB314409].mkv` is already marked seen in the task anime2 at 2022-05-31 23:33
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Honzuki no Gekokujou - 34 (1080p) [361A7367].mkv` by seen plugin because entry with title `[SubsPlease] Honzuki no Gekokujou - 34 (1080p) [361A7367].mkv` is already marked seen in the task anime2 at 2022-05-30 23:20
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Otome Game Sekai wa Mob ni Kibishii Sekai desu - 09 (1080p) [B9B04B93].mkv` by seen plugin because entry with title `[SubsPlease] Otome Game Sekai wa Mob ni Kibishii Sekai desu - 09 (1080p) [B9B04B93].mkv` is already marked seen in the task anime2 at 2022-05-29 23:39
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Kawaii dake ja Nai Shikimori-san - 07 (1080p) [CA7E54B2].mkv` by seen plugin because entry with title `[SubsPlease] Kawaii dake ja Nai Shikimori-san - 07 (1080p) [CA7E54B2].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Kakkou no Iinazuke - 06 (1080p) [C90ADCC1].mkv` by seen plugin because entry with title `[SubsPlease] Kakkou no Iinazuke - 06 (1080p) [C90ADCC1].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Spy x Family - 08 (1080p) [E1866698].mkv` by seen plugin because entry with title `[SubsPlease] Spy x Family - 08 (1080p) [E1866698].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Yatogame-chan Kansatsu Nikki S4 - 08 (1080p) [4EA5C260].mkv` by seen plugin because entry with title `[SubsPlease] Yatogame-chan Kansatsu Nikki S4 - 08 (1080p) [4EA5C260].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Ao Ashi - 08 (1080p) [21B0C543].mkv` by seen plugin because entry with title `[SubsPlease] Ao Ashi - 08 (1080p) [21B0C543].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Love All Play - 09 (1080p) [EBDD0D99].mkv` by seen plugin because entry with title `[SubsPlease] Love All Play - 09 (1080p) [EBDD0D99].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Dance Dance Danseur - 08 (1080p) [38C312D3].mkv` by seen plugin because entry with title `[SubsPlease] Dance Dance Danseur - 08 (1080p) [38C312D3].mkv` is already marked seen in the task anime1 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Aharen-san wa Hakarenai - 09 (1080p) [97D8A57D].mkv` by seen plugin because entry with title `[SubsPlease] Aharen-san wa Hakarenai - 09 (1080p) [97D8A57D].mkv` is already marked seen in the task anime2 at 2022-05-28 23:18
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Legend of the Galactic Heroes - Die Neue These - 35 (1080p) [C8C3D69C].mkv` by seen plugin because entry with title `[SubsPlease] Legend of the Galactic Heroes - Die Neue These - 35 (1080p) [C8C3D69C].mkv` is already marked seen in the task anime2 at 2022-05-26 23:54
2022-06-01 23:50:34 VERBOSE  task          anime3          REJECTED: `[SubsPlease] Mahoutsukai Reimeiki - 07 (1080p) [9356CBBD].mkv` by seen plugin because entry with title `[SubsPlease] Mahoutsukai Reimeiki - 07 (1080p) [9356CBBD].mkv` is already marked seen in the task anime2 at 2022-05-26 23:54
2022-06-01 23:50:34 VERBOSE  details       anime3          Summary - Accepted: 0 (Rejected: 17 Undecided: 33 Failed: 0)
  
### Additional information:

- FlexGet version: 3.3.14
- Python version: 3.8.10
- Installation method: pip install
- Using daemon (yes/no): no
- OS and version: Linux Lite 5.6, based on Ubuntu 20.04

**Comment by aidan-** [comment]
The rejected list is populated by the `remember_rejected` plugin which only tracks rejections by plugins that pass in `remember=True` or a value for `remember_time` arguments.  A quick look shows that these are the only plugins that populate those fields:
- `private_torrents`
- `content_filter`
- `content_size`
- `magnets`
- `only_new`
- `torrent_alive` 

Looking at the features of these plugins, they all require downloading and parsing of a `.torrent` file.  I would guess that the `remember_rejected` plugin is primarily for reducing the recurrence of the 'expensive' download and parsing operation in subsequent runs.

tldr; The `seen` plugin rejects don't get populated into the reject list.

**Comment by shinjidf** [comment]
Thank you @aidan- , now I understand. I can't access the flexget site now (certificate error), could you give a suggestion to use the "rejected" plugin?
